dashboard/app/views/levels/_admin.html.haml

```diff
@@ -36,7 +36,7 @@
         %li (Cannot edit)
 
     %br/
-    This level is in #{@level.script_levels.count} scripts:
+    This level is in #{@level.script_levels.to_a.count} scripts:
```

Comment:
This does appear to in fact prevent a DB dip. script_levels are already loaded and cached. The ActiveRecord count() method appears to trigger a database call -- SELECT count(*) from script_levels where...  even though the user_levels are loaded. to_a makes this an array count method of already loaded levels, not the ActiveRecord count. Adding the to_a does remove the database dip, clearly shown before vs after in log output and rack-mini-profiler output. This extra dip shows up now in development mode (not just levelbuilder) because the "Extra Links" section is visible in development mode. Agree this does not impact production. But this is noise that everyone doing perf debugging in the future would have to understand and ignore while doing perf analysis on their local system. This is a future perf debugging tax I am removing.